ObjcVoiceQuickstart/ViewController.m

```diff
@@ -618,8 +630,11 @@ - (void)provider:(CXProvider *)provider performAnswerCallAction:(CXAnswerCallAct
     self.audioDevice.enabled = NO;
     self.audioDevice.block();
     
+    __weak typeof(self) weakSelf = self;
     [self performAnswerVoiceCallWithUUID:action.callUUID completion:^(BOOL success) {
         if (success) {
+            __strong typeof(self) strongSelf = weakSelf;
+            [strongSelf.callKitProvider reportOutgoingCallWithUUID:action.callUUID connectedAtDate:[NSDate date]];
```

Comment:
Did we have a bug in our QS?

Reply:
Somehow it worked in the test app without having to do this. Need to compare the diff.

Reply:
After some debugging, it does not seem necessary to do this. Some intermittent call reporting issue was happening on my local device. Things started working after re-installing the app so reverting the change.